src/App.tsx

```diff
@@ -18,17 +19,31 @@ export default function App() {
 
 	const [playing, setPlaying] = useState(false);
 
+	/* Flags for clearing file name if parsing error is encountered */
+	const [markerParsingError, setMarkerParsingError] = useState<boolean>(false);
+	const [forceParsingError, setForceParsingError] = useState<boolean>(false);
+	const [error, setError] = useState<Error|null>(null);
```

Comment:
I think this would be better as just two state values, with `Error | null` for each file

Reply:
I'm trying to get this to work, but I'm having issues. If there are two separate error variables that can persist, if I upload an invalid marker file, then upload an invalid force file, the force file error doesn't override the marker file error, so it doesn't show what went wrong. Second, if I upload an invalid force file (let's say by accident), then upload a valid marker file, the force file error persists and keeps the error message visible.

Reply:
Having one error variable allows the other file upload button to clear the error, and having two boolean flags allows the "no file chosen" message to work independently for both buttons.

Reply:
I think it would be better to force the user to acknowledge the error, whether that means uploading the correct file or somehow clearing it, rather than implicitly dismissing it

Reply:
They are forced to address it. If they are intentionally trying to upload a file and it errors out, the only way to clear the error of the same file type is to upload a correct file. If they upload two erroneous files, then one correct one, they were alerted that the other was incorrect and no data is being displayed for that one, thus they are forced to upload the correct file to see their data. If they accidentally use the wrong button and it errors, it's more convenient that they can just upload using the correct button and it will work without blocking them or adding an extraneous UI button.